common-crafting.lic

```diff
@@ -106,9 +106,17 @@ module DRCC
   def get_crafting_item(name, bag, bag_items, belt)
     waitrt?
     if belt && belt['items'].find { |item| /\b#{name}/i =~ item || /\b#{item}/i =~ name }
```

Comment:
I have a terrible time with git....but you're correct there is a cleaner way to accomplish the goal here by removing the else on line 121. I edited the new code into the branch but the new code is:
`  def get_crafting_item(name, bag, bag_items, belt)
    waitrt?
    if belt && belt['items'].find { |item| /\b#{name}/i =~ item || /\b#{item}/i =~ name }
      if "You remove" == DRC.bput("untie my #{name} from my #{belt['name']}", "You remove", "Untie what")
          return
      end
    end
    command = "get my #{name}"
    command += " from my #{bag}" if bag_items && bag_items.include?(name)
    case DRC.bput(command, '^You get', '^You are already', '^What do you', '^What were you', 'You pick up', "can't quite lift it")
    when 'What do you', 'What were you'
      echo("You seem to be missing: #{name}")
      exit
    when "can't quite lift it"
      get_crafting_item(name, bag, bag_items, belt)
    end
  end`